sdk/js/src/utils/consts.ts

```diff
@@ -346,8 +346,8 @@ const MAINNET = {
     nft_bridge: undefined,
   },
   snaxchain: {
-    core: undefined,
-    token_bridge: undefined,
+    core: "0xc1BA3CC4bFE724A08FbbFbF64F8db196738665f4",
```

Comment:
Did these get added to the wormhole-sdk-ts repo?

Reply:
Yup. See [this PR](https://github.com/wormhole-foundation/wormhole-sdk-ts/pull/673).